views/partials/header.ejs

```diff
@@ -13,6 +13,10 @@
       href="http://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css"
```

Comment:
@thadakn1 I'm seeing here we have two different links for font-awesome. Do we need both? I don't think we do, and we should make sure that it works on the mobile side as well.

I have personally encountered bugs where the fa-icons show up on the mobile view on the desktop but not on an actual mobile phone, so it would be good to get a look at that too @paytonshaltis